src/main/java/org/quiltmc/asmr/processor/AsmrProcessorRunner.java

```diff
@@ -456,89 +442,65 @@ private void shiftRefCapturesFrom(List<AsmrReferenceCapture> sortedRefCaptures,
             Ref refA = refs.get(indexA);
             int[] pathPrefixA = refA.capture.pathPrefix();
 
-            bLoop:
             for (int indexB = indexA + 1; indexB < refs.size(); indexB++) {
                 Ref refB = refs.get(indexB);
                 int[] pathPrefixB = refB.capture.pathPrefix();
 
                 // check that the b prefix starts with the a prefix
-                if (pathPrefixB.length < pathPrefixA.length) {
-                    break bLoop;
+                int bLength = pathPrefixB.length;
+                int aLength = pathPrefixA.length;
+
+                if (bLength < aLength) {
+                    break;
                 }
-                for (int i = pathPrefixA.length - 1; i >= 0; i--) {
+
+                for (int i = 0; i < aLength; i++) {
                     if (pathPrefixB[i] != pathPrefixA[i]) {
-                        break bLoop;
+                        break;
```

Comment:
The `bLoop` label was there for a reason, it was to break out of this nested loop. Please add it back. I'm also using that label on the other breaks and continues to clarify which loop they are referring to.

Reply:
Huh, I don't know why I removed it here. I don't feel like it's necessary at the other places, but I'll add them back in.